internal/dag/extension_processor.go

```diff
@@ -84,7 +84,7 @@ func (p *ExtensionServiceProcessor) buildExtensionService(
 	ext *contour_api_v1alpha1.ExtensionService,
 	validCondition *contour_api_v1.DetailedCondition,
 ) *ExtensionCluster {
-	tp, err := timeoutPolicy(ext.Spec.TimeoutPolicy, p.ConnectTimeout)
+	tp, _, err := timeoutPolicy(ext.Spec.TimeoutPolicy, p.ConnectTimeout)
```

Comment:
should we be getting and using the cluster timeout policy here too? I don't see anywhere where it's being set otherwise

Reply:
😄  no problem, we've all done it. Updates look good!